app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -235,7 +237,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.updateAdmin(userId, body)
+        val actualResponse = apiService.updateAdmin(bearerToken, userId, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver